docs/getMessages.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
```

Comment:
```suggestion

set -euo pipefail

```

Reply:
@vvbandeira I will KIV this change, since it terminates due to python error of detecting repeated codes. 
@maliberty Are there anything I should take note for changing the duplicate warning/error numbers? e.g. chronological order within filename?